website/src/components/dev-ex-cards.tsx

```diff
@@ -0,0 +1,47 @@
+import { cn, Heading, InfoCard } from '@theguild/components';
+
+import boxSvg from './icons/box.svg';
+import checkmarksSvg from './icons/checkmarks.svg';
+import speedometerSvg from './icons/speedometer.svg';
+
+export function DevExCards(props: { className?: string }) {
+  return (
+    <section className={cn('px-4 py-6 sm:py-12 md:px-6 lg:py-24 xl:px-[120px]', props.className)}>
+      <Heading as="h3" size="md" className="text-balance text-center">
+        Your chance to fully use GraphQL.
+      </Heading>
+      <p className="mx-auto mt-4 max-w-[700px] text-center text-green-800">
+        Codegen enhances your GraphQL development with fully typed client and server code, generating robust,
+        error-resistant solutions in seconds
+      </p>
+      <ul className="mt-6 flex flex-row flex-wrap justify-center gap-2 md:mt-16 md:gap-6">
+        <InfoCard
+          as="li"
+          heading="Typed Queries, Mutations, and Subscriptions"
+          icon={<img src={boxSvg.src} alt="" />}
```

Comment:
why not use NextImage?
```suggestion
          icon={<NextImage src={boxSvg} alt="Box" />}
```

Reply:
yeah, you are right decoration images can use `alt=""` (didn't know that)